apps/dashboard/app/views/batch_connect/shared/_app_list.html.erb

```diff
@@ -1,7 +1,6 @@
-<div class="panel panel-<%= local_assigns.fetch(:style, "default") %>">
-  <div class="panel-heading"><%= title %></div>
-  <div class="list-group">
-
+<div class="card panel panel-<%= local_assigns.fetch(:style, "default") %>">
+  <div class="card-header text-muted"><%= title %></div>
```

Comment:
Do we want to mute this text here? Previously it wasn't. (I get that the sub categories are)

Reply:
Fixed in https://github.com/OSC/ondemand/pull/927/commits/0dcedb05342cde5fb842137de7aed47a517c90bb

Reply:
We definitely will remove the "text-muted" for the title!